app/src/main/java/com/example/jondhc/yoyo/Level1Activity.java

```diff
@@ -1,23 +1,288 @@
+        berry9.setOnTouchListener(new DragTouchListener());
+        //end of Adding drag and drop functionality to every berry.
+
+        //Switch-case for the quantity of fruits to be generated
+        switch (quantity) {
+            case 1:
+                berry4.setVisibility(View.VISIBLE);
+                break;
+            case 2:
+                berry6.setVisibility(View.VISIBLE);
+                berry2.setVisibility(View.VISIBLE);
+                break;
+            case 3:
+                berry4.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                break;
+            case 4:
+                berry1.setVisibility(View.VISIBLE);
+                berry4.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry9.setVisibility(View.VISIBLE);
+                break;
+            case 5:
+                berry1.setVisibility(View.VISIBLE);
+                berry3.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry9.setVisibility(View.VISIBLE);
+                break;
+            case 6:
+                berry1.setVisibility(View.VISIBLE);
+                berry2.setVisibility(View.VISIBLE);
+                berry3.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry8.setVisibility(View.VISIBLE);
+                break;
+            case 7:
+                berry1.setVisibility(View.VISIBLE);
+                berry2.setVisibility(View.VISIBLE);
+                berry4.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry7.setVisibility(View.VISIBLE);
+                berry9.setVisibility(View.VISIBLE);
+                break;
+            case 8:
+                berry1.setVisibility(View.VISIBLE);
+                berry2.setVisibility(View.VISIBLE);
+                berry4.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry7.setVisibility(View.VISIBLE);
+                berry8.setVisibility(View.VISIBLE);
+                berry9.setVisibility(View.VISIBLE);
+                break;
+            case 9:
+                berry1.setVisibility(View.VISIBLE);
+                berry2.setVisibility(View.VISIBLE);
+                berry3.setVisibility(View.VISIBLE);
+                berry4.setVisibility(View.VISIBLE);
+                berry5.setVisibility(View.VISIBLE);
+                berry6.setVisibility(View.VISIBLE);
+                berry7.setVisibility(View.VISIBLE);
+                berry8.setVisibility(View.VISIBLE);
+                berry9.setVisibility(View.VISIBLE);
```

Comment:
use an array list to perform this kind of tasks on a loop, will use a bit more memory but code is much simplier